code/formatters.py

```diff
@@ -112,6 +113,13 @@ def formatter_function(i, word, _):
         first_vs_rest(lambda w: w, lambda w: w.capitalize()),
     ),
     "PUBLIC_CAMEL_CASE": (NOSEP, every_word(lambda w: w.capitalize())),
+    "STRIKETHROUGH": (
+        NOSEP,
+        first_vs_rest(
+            lambda w: strikethrough_word(w),
+            lambda w: strikethrough_word(" ") + strikethrough_word(w),
```

Comment:
As a minor optimisation, we could probably store the result of `strikethrough_word(" ")` as a variable so that it doesn't get rerun on every lambda call.

I don't think it really matters as much though from a performance perspective and it probably reduces understandability.